_includes/tutorials/session-windows/ksql/markup/dev/create-inputs.adoc

```diff
@@ -0,0 +1,4 @@
+The first thing we'll do is create and populate the stream of click data. An important characteristic of this data is the _timestamp_ because this is what drives the session window. KSQL can use either the Kafka message timestamp, or a field from the message value as the timestamp. In this example we'll use the latter—the _event time_ as stored in the `timestamp` field of the message value.
```

Comment:
```suggestion
The first thing we'll do is create and populate the stream of click data. An important characteristic of this data is the _timestamp_ because this is what drives the session window. ksqlDB can use either the Kafka message timestamp, or a field from the message value as the timestamp. In this example we'll use the latter—the _event time_ as stored in the `timestamp` field of the message value.
```